schedule/yast/autoyast_reinstall.yaml

```diff
@@ -0,0 +1,25 @@
+name:           autoyast_reinstall
+description:    >
+    Parent job produces autoyast profile after successful completion. 
+    This test uses generated profile to do autoyast installation.
+schedule:
+    - installation/isosize
+    - installation/bootloader
```

Comment:
We have this test enabled for 4 architectures, but looks like we can simply use our `installation/bootloader_start` wrapper.